app/app.rb

```diff
@@ -0,0 +1,62 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative '../data_mapper_setup'
+require_relative "./models/user.rb"
+
+
+class Chitter < Sinatra::Base
+	register Sinatra::Flash
+  enable :sessions
+  set :session_secret, 'super secret'
+
+
+  get '/' do
+  	redirect "users/new"
+  end
+
+	get '/sessions/new' do
+    erb :'sessions/new'
+  end
```

Comment:
Lint/BlockAlignment: `end` at 21, 2 is not aligned with `get '/sessions/new' do` at 19, 1.